### PR TITLE
use array_key_exists instead of isset for __get and hasAttribute

### PR DESCRIPTION
**Pull request by Faryshta**
http://php.net/manual/en/function.array-key-exists.php `isset` returns null if an array key has a null value, that means sometimes the key might exists on an array but `isset` will return false

It was previously being used on the `__get` method but it was used after calling `isset` making calling `isset` redundant.

### CODE REVIEW

**Comment by samdark** [comment]
As you can see, it's breaking everything: https://travis-ci.org/yiisoft/yii2/jobs/49158411

File: framework/db/BaseActiveRecord.php [hunk 1]
```diff
@@ -432,7 +433,9 @@ public function hasAttribute($name)
      */
     public function getAttribute($name)
     {
-        return isset($this->_attributes[$name]) ? $this->_attributes[$name] : null;
+        return isset($this->_attributes[$name])
```
Comment (Faryshta) [inline framework/db/BaseActiveRecord.php]: error at this line

**Comment by Faryshta** [comment]
i found the problem and did another commit, how can i update this pull?

**Comment by samdark** [comment]
Commit to the same branch.

**Comment by Faryshta** [comment]
I feel happy now

**Comment by mikehaertl** [comment]
Only a sidenote: This commit here accidentally added a vim .swp file to the repo:

https://github.com/Faryshta/yii2/commit/025bc05885f8f9a188d65400865a4d37ef2d123f

Even if you remove this with another commit, this file will forever live in the repo history. So maybe it's cleaner to either create a fresh new PR or reset the commits, do a new clean commit and force push the changes to your branch.

**Comment by samdark** [comment]
We can squash it, not problem.

**Comment by Faryshta** [comment]
whatever you prefer, i don't know if there is a command to erase the .swp file from the commits.

**Pull request closed without merging**